# Fix linker errors building with is_multi_dll_chrome=false

--- a/headless/BUILD.gn
+++ b/headless/BUILD.gn
@@ -406,7 +406,7 @@ component("headless") {
     deps += [ "//components/os_crypt" ]
   }
 
-  if (is_component_build) {
+  if (true) {
     sources += [
       "lib/browser/headless_content_browser_client.cc",
       "lib/browser/headless_content_browser_client.h",
@@ -480,7 +480,7 @@ component("headless") {
 
 # Headless renderer is a convenience library for non component builds that
 # includes headless classes that depend on the renderer.
-if (!is_component_build) {
+if (false) {
   static_library("headless_renderer") {
     sources = [
       "lib/browser/headless_web_contents_impl.cc",
